packetbeat/docs/index.asciidoc

```diff
@@ -19,49 +19,71 @@ include::../../libbeat/docs/version.asciidoc[]
 
 include::./overview.asciidoc[]
 
+pass::[<?edit_url https://github.com/elastic/beats/edit/master/packetbeat/docs/gettingstarted.asciidoc ?>]
 include::./gettingstarted.asciidoc[]
 
+
+pass::[<?edit_url https://github.com/elastic/beats/edit/master/packetbeat/docs/command-line.asciidoc ?>]
 include::./command-line.asciidoc[]
 
+pass::[<?edit_url https://github.com/elastic/beats/edit/master/libbeat/docs/shared-directory-layout.asciidoc ?>]
 include::../../libbeat/docs/shared-directory-layout.asciidoc[]
 
+pass::[<?edit_url https://github.com/elastic/beats/edit/master/packetbeat/docs/upgrading.asciidoc ?>]
 include::./upgrading.asciidoc[]
 
+pass::[<?edit_url https://github.com/elastic/beats/edit/master/packetbeat/docs/configuring-howto.asciidoc ?>]
 include::./configuring-howto.asciidoc[]
 
+pass::[<?edit_url https://github.com/elastic/beats/edit/master/packetbeat/docs/packetbeat-filtering.asciidoc ?>]
 include::./packetbeat-filtering.asciidoc[]
 
+pass::[<?edit_url https://github.com/elastic/beats/edit/master/libbeat/docs/shared-config-ingest.asciidoc ?>]
 include::../../libbeat/docs/shared-config-ingest.asciidoc[]
 
+pass::[<?edit_url https://github.com/elastic/beats/edit/master/packetbeat/docs/packetbeat-geoip.asciidoc ?>]
 include::./packetbeat-geoip.asciidoc[]
 
+pass::[<?edit_url https://github.com/elastic/beats/edit/master/libbeat/docs/shared-logstash-config.asciidoc ?>]
```

Comment:
I think here it should point to: https://github.com/elastic/beats/edit/master/packetbeat/docs/shared-logstash-config.asciidoc